src/ui/screens/settings/ViewSubscriptionManagement/ViewSubscriptionManagement.qml

```diff
@@ -159,7 +159,9 @@ VPNViewBase {
                     VPNSubscriptionData.planCurrency,
                     VPNSubscriptionData.planAmount,
                 ),
-                type: (Qt.locale().name === "en_US" || Qt.locale().name === "en_CA" ) && VPNFeatureList.get("bundleUpgrade").isSupported ? "text-upgrade" : "text",
+                type: VPNSubscriptionData.bundleUpgradeAvailable && VPNFeatureList.get("bundleUpgrade").isSupported
```

Comment:
This is confusing. Why part of the conditions are in c++ and part in Js? I would do everything here.

Reply:
Since we also introduce a check to see if a user is already a bundle subscriber with PR #4474 I thought it might better idea to not move this logic to QML. Are you also OK with moving `VPNFeatureList.get("bundleUpgrade").isSupported` to the C++ layer instead of moving everything to QML?

Reply:
Moved the check to QML.